packages/apps-service/src/modules/apps/resolver.ts

```diff
@@ -1,192 +1,198 @@
+/* eslint-disable no-param-reassign */
+/* eslint-disable @typescript-eslint/naming-convention */
+/* eslint-disable import/no-cycle */
+/* eslint-disable no-underscore-dangle */
+/* eslint-disable @typescript-eslint/no-unused-vars */
```

Comment:
What's the point of using eslint if the rules are disabled?

Reply:
updated